sdk-api-src/content/msi/nf-msi-msisetinternalui.md

```diff
@@ -165,7 +165,16 @@ If combined with the <b>INSTALLUILEVEL_BASIC</b> value, the installer shows simp
 </td>
 <td width="60%">
 If this value is combined with the <b>INSTALLUILEVEL_NONE</b> value, the installer displays only the dialog boxes used for source resolution. No other dialog boxes are shown. This value has no effect if the UI level is not <b>INSTALLUILEVEL_NONE</b>. It is used with an external user interface designed to handle all of the UI except for source resolution. In this case, the installer handles source resolution.
-
+ 
+</td>
+</tr>
+<td width="40%"><a id="INSTALLUILEVEL_UACONLY"></a><a id="installuilevel_uaconly"></a><dl>
+<dt><b>INSTALLUILEVEL_UACONLY</b></dt>
+</dl>
+</td>
+<td width="60%">
+When specified will provide a completely silent installation, but will still prompt for elevation if required.  
```

Comment:
When provided *with INSTALLLUILEVEL_NONE* ...

I don't think the flag has any use beyond the affect it has with NONE.

Comment in header for the value is:
```
INSTALLUILEVEL_UACONLY = 0x200, // show UAC prompt even if quiet
```